db/migrate/20160331232622_devise_create_users.rb

```diff
@@ -1,42 +0,0 @@
-class DeviseCreateUsers < ActiveRecord::Migration
```

Comment:
@dvanetten03 how come this file needs to be deleted? Do we have two of the same migration file in the app?